File: day13/code.py
#Leetcode Solution

# class Solution:
#     def maxProduct(self, nums: List[int]) -> int:
#         nums.sort()
#         return (nums[-1]-1) * (nums[-2]-1)

def maxProduct(nums):
    nums.sort()
    return (nums[-1]-1) * (nums[-2]-1)
    
n=int(input("Enter length of list: "))
nums=[]
for i in range(n):
    nums.append(int(input("Enter score: ")))    
result=maxProduct(nums)
print(result)

File: day51/code.cpp
// Leetcode Solution

// class Solution {
// public:
//     int scoreOfString(string s) {
//         int score = 0;
//         int n = size(s);
//         for (int i=1; i<n;i++){
//             int diff = int(s[i-1]) - int(s[i]);
//             if (diff<0){
//                 score += (diff*(-1));
//             }
//             else{
//                 score += diff;
//             }
//         }
//         return score;
//     }
// };